isvcs/container.go

```diff
@@ -879,7 +883,7 @@ func (svc *IService) setHealthStatus(result error, currentTime int64) {
 		"isvc": svc.Name,
 	})
 
-	if healthStatus, found := svc.healthStatuses[DEFAULT_HEALTHCHECK_NAME]; found {
+	if healthStatus, found := svc.healthStatuses[healthCheckIndex]; found {
 		if result == nil {
 			if healthStatus.Status != "passed" && healthStatus.Status != "unknown" {
 				log.Info("Internal service checked in healthy")
```

Comment:
This Info message needs to identify the healthcheck that passed, and probably should be reworded a bit since it's not the entire "Internal service" that's healthy - just one of possibly several checks that passed